src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.hpp

```diff
@@ -154,8 +159,7 @@ class FlightTaskAutoFollowTarget : public FlightTask
 
 	void point_gimbal_at(float xy_distance, float z_distance);
 	matrix::Vector2f calculate_offset_vector_filtered(matrix::Vector3f vel_ned_est);
-	matrix::Vector3f calculate_target_position_filtered(matrix::Vector3f pos_ned_est, matrix::Vector3f vel_ned_est,
-			matrix::Vector3f acc_ned_est);
+	matrix::Vector3f calculate_target_position_filtered(matrix::Vector3f pos_ned_est, matrix::Vector3f vel_ned_est);
```

Comment:
additionally on naming .. this is more a pedantic request.. but it is nice when method names are not spoken like yoda :) e.g. `update_filtered_target_pose`

Reply:
I will keep Yoda away ;)